habitat-baselines/habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -268,6 +268,7 @@ def _init_train(self, resume_state=None):
         self._ppo_cfg = self.config.habitat_baselines.rl.ppo
 
         observations = self.envs.reset()
+        observations = self.envs.post_step(observations)
```

Comment:
Can you raise an exception in the VER trainer to explain that batched rendering doesn't work with VER for now?